src/features/HelpAndKnowledge/SolutionsHelp/SolutionDetails/Solutions/Generic/_components/ContractorModal/index.tsx

```diff
@@ -48,7 +56,31 @@ const ContractorModal = ({
         mode={mode}
         closeModal={closeModal}
         contractor={contractor}
+        setSubmitForm={setSubmitContractorForm}
```

Comment:
Here is where you need to alter how you are passing down the setter

```suggestion
        setSubmitForm={() => setSubmitContractorForm}
```